packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -605,8 +606,19 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
     this.addEventListener('copy', this.handleCopy);
     this.addEventListener('cut', this.handleCut);
     this.addEventListener('paste', this.handlePaste);
+    this.addEventListener('selectionChanged', this.handleSelectionChanged);
   }
 
+  /**
+   * Clears the disabled property on the people picker when used in single mode.
+   */
+  private readonly handleSelectionChanged = () => {
```

Comment:
I'm confused by this. 

we have code in the `renderInput` that looks at the `disabled` property to determine if the fluent-text-field should disabled or not. 

IMO we should be ensuring that we set the disabled property to the correct value rather than making DOM manipulations in this manner